src/coreclr/jit/importer.cpp

```diff
@@ -13524,8 +13566,17 @@ void Compiler::impImportBlockCode(BasicBlock* block)
                 op1 = impPopStack().val; // operand to be shifted
                 impBashVarAddrsToI(op1, op2);
 
+#ifdef TARGET_LOONGARCH64
```

Comment:
Please remove this. There should be no architecture-specific code in the importer.

Reply:
had removed.
Thanks